scalyr_agent/builtin_monitors/windows_event_log_monitor.py

```diff
@@ -754,6 +774,14 @@ def log_event(self, event):
         #        => '{"entry": {"text": "bar", "#text": "foo"}}'
         event_json = _strip_xmltodict_prefixes(event_json)
 
+        # FormatMessage will replace insertion and parameter strings in the event message,
+        # however it will not not replace parameter strings in the event data.
```

Comment:
duplicate "not", not sure if this can be considered as double negative :D

Reply:
Oops, will fix, thank you!